tests/pkunit_test.py

```diff
@@ -17,6 +17,20 @@ def test_assert_object_with_json():
         pkunit.assert_object_with_json("assert1", {"b": 1})
 
 
+def test_pkfail_output():
+    from pykern.pkunit import pkexcept, pkne, pkeq
+
+    with pkexcept("expecting ValueError but will get=KeyError"):
```

Comment:
This should check that you see the KeyError message itself, too.

Reply:
Sorry. I'll fix